Dockerfile

```diff
@@ -17,17 +17,17 @@
 # Now you can connect to:
 #    http://172.17.0.1:8891
 #
-FROM ubuntu:14.04
+FROM ubuntu:16.04
 MAINTAINER pietro.delsante_AT_gmail.com
 ENV DEBIAN_FRONTEND noninteractive
 EXPOSE 8891
 
 RUN apt-get update && \
-    apt-get -y install curl && \
+    apt-get -y install curl sudo && \
```

Comment:
Since the next line use sudo and Ubuntu 16.04 image did not include sudo package, I just added it.
If next line sudo is not necessary, I believe that sudo should be removed.